components/expenses/PayoutMethodSelect.js

```diff
@@ -98,7 +106,15 @@ class PayoutMethodSelect extends React.Component {
   getOptions = memoizeOne(payoutMethods => {
     const { formatMessage } = this.props.intl;
     const groupedPms = groupBy(payoutMethods, 'type');
-    return Object.values(PayoutMethodType).map(pmType => ({
+    const pmTypes = Object.values(PayoutMethodType).filter(type => {
+      if (type === PayoutMethodType.BANK_ACCOUNT && !this.props.collective.host.transferwise) {
+        return false;
+      } else {
+        return true;
+      }
+    });
```

Comment:
`this.props.collective.host` can be null. Also because entries are grouped by type we don't really need to go through the entire object:

```suggestion
    if (!this.props.collective.host?.transferwise) {
      delete groupedPms[PayoutMethodType.BANK_ACCOUNT]
    }
```

Reply:
Nice!

Reply:
Not exactly, because we would still display the `+ Create new Wire Transfer` option.

Reply:
True :+1: